nem-docker/boot.sh

```diff
@@ -13,36 +13,35 @@ docker ps -a | grep mynem_container > /dev/null && docker rm mynem_container
 
 
 # determine which custom configs to mount
-
-set -x
```

Comment:
why did you delete this ?

Reply:
dont think debug logs are needed in customer scripts but can add it back.

Reply:
would prefer to keep it

Reply:
any reason not to use set -ex ?

Reply:
``set -eu`` is already at the top of the file.